src/DrawerPanel.tsx

```diff
@@ -16,7 +16,14 @@ export interface DrawerPanelEvents {
   onKeyUp?: React.KeyboardEventHandler<HTMLDivElement>;
 }
 
-export interface DrawerPanelProps extends DrawerPanelEvents {
+export type DrawerPanelAccessibility = Pick<
+  React.DialogHTMLAttributes<HTMLDivElement>,
+  'aria-labelledby' | 'aria-describedby'
```

Comment:
Why not `aria-*`?

Reply:
I could add that, wasn't sure if the current restrictions were in place for some reason I wasn't aware of :)

Reply:
done